Project1/index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<body>
+
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <link href="https://fonts.googleapis.com/css?family=Bitter" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css?family=Bitter|Bree+Serif|EB+Garamond:600|PT+Serif" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css?family=Bitter|Bree+Serif|PT+Serif" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css?family=Bitter|PT+Serif" rel="stylesheet">
+        <link rel="stylesheet" href="styles.css">
+        <title>Document</title>
+    </head>
+
+    <figure>
+        <img src="images/maldivesoctopus.jpg" alt="an octopus swims in the ocean.">
+        <figcaption>This is an octopus.</figcaption>
+    </figure>
+
+    <h1>
+        <strong>Yes, the Octopus Is Smart as Heck. But Why?</strong>
+    </h1>
+
+    <h2>It has eight arms, three hearts — and a plan. Scientists aren’t sure how the cephalopods got to be so
+        intelligent. By Carl Zimmer Nov. 30, 2018
+    </h2>
+
+    <p>
+        To demonstrate how smart an octopus can be, Piero Amodio points to a YouTube video. It shows an octopus pulling
+        two halves of a coconut shell together to hide inside. Later the animal stacks the shells together like nesting
+        bowls — and carts them away.“It suggests the octopus is carrying these tools around because it has some
+        understanding they
+        may be useful in the future,” said Mr. Amodio, a graduate student studying animal intelligence at the
+        University of
+        Cambridge in Britain.</p>
+
+    <aside>
+        <blockquote>
+            <p class="pullquote">
+                <i>“It suggests the octopus is carrying these tools around because it has some
```

Comment:
The [`i` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/i) is reserved for "technical terms, foreign language phrases, or fictional character thoughts," so you can delete it here:

```suggestion
                “It suggests the octopus is carrying these tools around because it has some
```